### PR TITLE
Patch specification and client for webauth route

### PR DESCRIPTION
**Pull request by armandmgt**

### CODE REVIEW

File: .swagger-codegen-ignore [hunk 1]
```diff
@@ -21,3 +21,5 @@
 #docs/*.md
 # Then explicitly reverse the ignore rule for a single file:
 #!docs/README.md
+
+Rakefile
```
Comment (nakhli) [inline .swagger-codegen-ignore]: 👍

File: budgea_openapi.json [hunk 2]
```diff
@@ -21224,49 +21224,49 @@
         "parameters": [
           {
             "required": false,
-            "in": "formData",
```
Comment (nakhli) [inline budgea_openapi.json]: Could please re-explain briefly the rationale behind this change `formData` => `query`?

Comment (armandmgt) [reply budgea_openapi.json]: The method is `get` so it makes no sense to send formData 🤷‍♂

**Comment by armandmgt** [comment]
I thought we should not rely on the new route until they confirm it will be added to their spec
Do you want to use it anyway right now ? @nakhli

**Comment by nakhli** [comment]
> I thought we should not rely on the new route until they confirm it will be added to their spec
> Do you want to use it anyway right now ? @nakhli

yeah let's do this and wait for they answer. merci beaucoup ! good job!

**Pull request merged**